integration-tests/suites/k8s/base.go

```diff
@@ -0,0 +1,189 @@
+package k8s
```

Comment:
Thanks for moving things from `k8s_namespace.go` to here.

Reply:
This file mirrors `integration-tests/suites/base.go`, we can consider moving more things from these files to a different package in a follow up PR, since setting up the testing environment for k8s is not the focus of this one.

Reply:
Doing it in a follow up PR is fine.